testing/run-scripts/run_monitor.sh

```diff
@@ -0,0 +1,18 @@
+# Boot monitoring servers and metrics.
+
+if netstat -lnt | awk '$6 == "LISTEN" && $4 ~ ".8000"' >/dev/null; then
+  sh ./serve-zork-metrics.sh
+fi
+
+if ! docker ps -a | grep prom/prometheus >/dev/null; then
```

Comment:
I think it would be better to give the container a name, e.g. `uproxy-prometheus` and check for that. The user could have other Prometheus containers running.